Portfolio/index.css

```diff
@@ -111,32 +116,174 @@ a {
     text-align: center;
 }
 
-.about{
+.about {
     display: flex;
     flex-direction: column;
     text-align: center;
     justify-content: center;
     padding: 2rem;
 }
-.about-title{

```

Comment:
Consider uncommenting the border property for footer to have a visual separation from surrounding elements